pymilvus/orm/iterator.py

```diff
@@ -37,7 +38,6 @@
     MILVUS_LIMIT,
     OFFSET,
     PARAMS,
-    PRINT_ITERATOR_CURSOR,
```

Comment:
Please remove log=logging.getLogger(__name__) in L55

and  LOGGER.setLevel(logging.INFO) in L51

Reply:
it's a separate file and I suppose I need to set up log as started?
log = logging.getLogger("pymilvus")
log.setLevel(logging.INFO)
changed at line 50~51

Reply:
`pymilvus/orm/iterator.py` is part of the source codes, so don't need to getLogger from pymilvus, just use getLogger(__name__) is enough, and donot setLevel in source code, this'll break settings outside of PyMilvus(such as log settings in examples)

Loggers for examples needs to getLogger("pymilvus")